src/transformers/models/xlm_prophetnet/modeling_xlm_prophetnet.py

```diff
@@ -876,11 +876,7 @@ def forward(
     ):
         batch_size, ngram_sequence_length, hidden_size = hidden_states.size()
 
-        assert list(hidden_states.size()) == [
-            batch_size,
-            ngram_sequence_length,
-            hidden_size,
-        ], (
+        assert list(hidden_states.size()) == [batch_size, ngram_sequence_length, hidden_size], (
```

Comment:
Same as before.

Reply:
Done.

Reply:
Black is not happy about this the old code it forces me to use this new layout.

I tried to revert this.